src/functional_test_runner/cli.js

```diff
@@ -21,10 +20,8 @@ cmd
   .parse(process.argv);
 
 if (!cmd.config) {
-  console.log('');
-  console.log('  --config is a required parameter');
-  console.log('');
-  process.exit(1);
+  const [project = 'functional'] = cmd.args;
+  cmd.config = resolveConfigPath(`test/${project}/config.js`);
 }
```

Comment:
Ah, i was going through each commit to understand changes, and this file changed more after [this commit](https://github.com/elastic/kibana/pull/11848/commits/b2ac4c26593a1947c94f0168191fe8123ff74122). help me understand why?